gitax/Classes/GTXPendingVCSObjects.xpo

```diff
@@ -69,11 +72,11 @@ Formatversion: 1
         #        if(!controlable)
         #            continue;
         #
-        #        if(versionControl.allowCreate(controlable))
-        #        {
-        #            //do something if it isn't in version control
-        #            info(strfmt("%1\t%2",pNode.treeNodeType(), pNode.treeNodeName()));
-        #        }
+        #        //if(versionControl.allowCreate(controlable))
```

Comment:
Personally, I don't need to be able to see history or pending items in Morphx. I can use a git client for that. Also don't need checkin or undo checkin. The main requirement for me is MorphX/AX being able to sync with what's on disk.